### PR TITLE
feat: Create tool for reading block files

### LINKED ISSUE #13921: Create a block reader to read `.blk` files and output in a human-readable way

### PR DESCRIPTION
**Pull request by mhess-swl**
Closes #13921

### CODE REVIEW

File: hedera-node/cli-clients/src/main/java/com/hedera/services/cli/block/BlockCommand.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+package com.hedera.services.cli.block;
+
+import com.swirlds.cli.PlatformCli;
+import com.swirlds.cli.utility.AbstractCommand;
+import com.swirlds.cli.utility.SubcommandOf;
+import picocli.CommandLine;
+
+/**
+ * A collection of operations on for blocks (a.k.a. the block stream)
```
Comment (thomas-swirlds-labs) [inline hedera-node/cli-clients/src/main/java/com/hedera/services/cli/block/BlockCommand.java:25]: nit: doesn't read well. Maybe "A collection of operations on block files" ?

**Review by thomas-swirlds-labs — APPROVED** [review]
LGTM. Thanks @mhess-swl 👍

File: hedera-node/cli-clients/src/main/java/com/hedera/services/cli/block/BlockCommand.java (hunk 1 above)
Comment (mhess-swl) [reply hedera-node/cli-clients/src/main/java/com/hedera/services/cli/block/BlockCommand.java]: Fixed

**Comment by Nana-EC** [comment]
Pinging @mattp-swirldslabs and @AlfredoG87 as an FYI

**Comment by mhess-swl** [comment]
For now, we'll go forward getting the block stream MVP to work before fixing all the tests. Merging now.

**Pull request merged**